internal/builders/go/pkg/provenance.go

```diff
@@ -18,13 +18,13 @@ import (
 	"context"
 	"encoding/hex"
 	"fmt"
+	"github.com/slsa-framework/slsa-github-generator/signing"
```

Comment:
Move this import to the 3p block below. This block is stdlib packages.

Reply:
Thanks! Took care of it.